tools/lan_choose_character_kart/asm/charkartdb.inc

```diff
@@ -0,0 +1,88 @@
+.equ BabyMarioDB, 1
+.equ BabyLuigiDB, 2
+.equ ParatroopaDB, 3
+.equ KoopaDB, 4
+.equ PeachDB, 5
+.equ DaisyDB, 6
+.equ MarioDB, 7
+.equ LuigiDB, 8
+.equ WarioDB, 9
+.equ WaluigiDB, 10
+.equ YoshiDB, 11
+.equ BirdoDB, 12
+.equ DonkeyKongDB, 13
+.equ DiddyKongDB, 14
+.equ BowserDB, 15
+.equ BowserJrDB, 16
+.equ ToadDB, 17
+.equ ToadetteDB, 18
+.equ KingBooDB, 19
+.equ PeteyPiranhaDB, 20
+
+# Matches order of the Character Selection screen
+.equ MarioOrder, 0
+.equ LuigiOrder, 1
+.equ PeachOrder, 2
+.equ DaisyOrder, 3
+.equ YoshiOrder, 4
+.equ BirdoOrder, 5
+.equ BabyMarioOrder, 6
+.equ BabyLuigiOrder, 7
+.equ ToadOrder, 8
+.equ ToadetteOrder, 9
+.equ KoopaOrder, 10
+.equ ParatroopaOrder, 11
+.equ DonkeyKongOrder, 12
+.equ DiddyKongOrder, 13
+.equ BowserOrder, 14
+.equ BowserJrOrder, 15
+.equ WarioOrder, 16
+.equ WaluigiOrder, 17
+.equ PeteyPiranhaOrder, 18
+.equ KingBooOrder, 19
+
+
+.equ RedFireDB, 0
+.equ DKJumboDB, 1
+.equ TurboYoshiDB, 2
+.equ KoopaDasherDB, 3
+.equ HeartCoachDB, 4
+.equ GooGooBuggyDB, 5
+.equ WarioCarDB, 6
+.equ KoopaKingDB, 7
+.equ GreenFireDB, 8
+.equ BarrelTrainDB, 9
+.equ TurboBirdoDB, 10
+.equ ParaWingDB, 11
+.equ BloomCoachDB, 12
+.equ RattleBuggyDB, 13
+.equ WaluigiRacerDB, 14
+.equ BulletBlasterDB, 15
+.equ ToadKartDB, 16
+.equ ToadetteKartDB, 17
+.equ BooPipesDB, 18
+.equ PiranhaPipesDB, 19
+.equ ParadeKartDB, 20
+
+# Matches order of karts on Character Selection screen, from light to heavy
+.equ GooGooBuggyOrder, 0
+.equ RattleBuggyOrder, 1
+.equ KoopaDasherOrder, 2
+.equ ParaWingOrder, 3
+.equ BarrelTrainOrder, 4
+.equ BulletBlasterOrder, 5
+.equ ToadKartOrder, 6
+.equ ToadetteKartOrder, 7
+.equ RedFireOrder, 8
+.equ GreenFireOrder, 9
+.equ HeartCoachOrder, 10
+.equ BloomCoachOrder, 11
+.equ TurboYoshiOrder, 12
+.equ TurboBirdoOrder, 13
+.equ WaluigiRacerOrder, 14
+.equ WarioCarOrder, 15
+.equ DKJumboOrder, 16
+.equ KoopaKingOrder, 17
+.equ PiranhaPipesOrder, 18
+.equ BooPipesOrder, 19
+.equ ParadeKartOrder, 20
```

Comment:
The assembler seems to be throwing some warnings due to some of these `.inc` files missing a trailing `\n` character:

```
.../mkdd-extender/tools/lan_choose_character_kart/asm/./fielddefinitions.inc: Assembler messages:
.../mkdd-extender/tools/lan_choose_character_kart/asm/./fielddefinitions.inc: Warning: end of file not at end of a line; newline inserted
```